Vagrantfile

```diff
@@ -208,4 +208,11 @@ Vagrant.configure('2') do |config|
     h.enable_virtualization_extensions = true
     h.linked_clone = true
   end
+
+  # Libvirt/KVM settings
+  config.vm.provider 'libvirt' do [lv]
```

Comment:
Just at a glance, shouldn't this be `config.vm.provider 'libvirt' do |lv|`? (pipes, not squares)

Reply:
that might very well be the case, I might've miss-typed or didn't notice it needed pipes... Like I said, wait with adding this until it has been tested :)

Reply:
Yeah, I think you are right here, I might've broken the vagrant file with this change. I'll make a quick update...

Reply:
Please see #1491 for a fix @swalkinshaw

Reply:
Thank you for finding this @talss89 :+1: